users/views.py

```diff
@@ -22,13 +20,6 @@ class LoginUserView(LoginView):
     def get_success_url(self: "LoginUserView") -> str:
         return reverse_lazy("home-page")
 
-    def form_valid(self: "LoginUserView", form: Form) -> HttpResponse:
-        valid = super().form_valid(form)
-        email, password = form.cleaned_data.get("email"), form.cleaned_data.get("password1")
```

Comment:
@yanazPL your task was to add username, not to change login functionality. Revert.